src/components/BookItem.js

```diff
@@ -15,8 +15,15 @@ const BookItem = (props) => {
 
   return (
     <li className="book">
+      <p>{book.category}</p>
       <h2>{book.title}</h2>
-      <button type="button" onClick={removeBookfromStore}>Delete</button>
+      <div className="buttons">
+        <button type="button">Comments</button>
+        |
+        <button type="button" onClick={removeBookfromStore}>Remove</button>
+        |
+        <button type="button">Edit</button>
+      </div>
     </li>
```

Comment:
- [ ] There some missing part which is in the design check on [Zeplin]()
![zp](https://user-images.githubusercontent.com/44745855/145434135-006c82e7-e989-4d41-9efa-d1caa6e437fa.png)

There are some features that are not yet implemented that you should add them as a placeholder